src/data/markdown/docs/30 xk6-browser/01 Getting started/03 Running xk6-browser.md

```diff
@@ -0,0 +1,227 @@
+
+  <Blockquote mod="note" title="">
+
+  To provide rough compatibility with Playwright API, xk6-browser API is also being converted from synchronous to asynchronous. `page.goto()` is now asynchronous so `.then()` is used to deal with the asynchronous nature of the operation. 
+
+  </Blockquote>
+
+3. Then, run xk6-browser on your terminal with this command:
+
+  <CodeGroup labels={["CLI"]}>
+
+  ```bash
+  $ ./xk6-browser run script.js
+  ```
+
+  </CodeGroup>
+
+  <Blockquote mod="note" title="">
+
+  The `./` prefix tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on the Windows `cmd.exe` shell. On PowerShell, specify `.\xk6-browser` instead.
+
+  If you installed xk6-browser with a system package, or placed the binary in a directory that's part of your `$PATH` environment variable, you can omit the `./` or `.\` prefixes.
+
+  </Blockquote>
+
+## Interact with elements on your webpage
+
+You can use `page.locator()` and pass in the element's selector you want to find on the page. `page.locator()` will create and return a [Locator](https://k6.io/docs/javascript-api/xk6-browser/locator/) object, which you can later use to interact with the element.
+
+<Blockquote mod="note" title="">
+
+You can also use `page.$()` instead of `page.locator()`. You can find the differences between `page.locator()` and `page.$` in the [Locator API documentation](https://k6.io/docs/javascript-api/xk6-browser/locator/).
+
+</Blockquote>
+
+<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+
+```javascript
+import { chromium } from 'k6/x/browser';
+
+export default function () {
+  const browser = chromium.launch({ headless: false });
+  const page = browser.newPage();
+
+  page
+    .goto('https://test.k6.io/my_messages.php')
+    .then(() => {
+      // Enter login credentials
+      page.locator('input[name="login"]').type('admin');
+      page.locator('input[name="password"]').type('123');
+
+      page.screenshot({ path: 'screenshot.png' });
+    })
+    .finally(() => {
+      page.close();
+      browser.close();
+    });
+}
+```
+
+</CodeGroup>
+
+The preceding code creates and returns a Locator object with the selectors for both login and password passed as arguments. 
+
+Within the Locator API, various methods such as `type()` can be used to interact with the elements. The `type()` method types a text to an input field. 
+
+## Asynchronous operations
+
+As explained previously, k6 operations are synchronous. However, Playwright operations are asynchronous in nature and since xk6-browser aims to provide rough compatibility with Playwright, some of our operations are now asynchronous.
```

Comment:
It might be good to mention that we're not just transitioning to async to follow Playwright's lead, but because it's a better fit for interacting with the browser.

Also, let's stick to a single way of referring to functions, or methods in this case, instead of "operations", which is a bit ambiguous in this context.

```suggestion
As explained previously, the k6 API is synchronous. However, since many browser operations happen asynchronously, and in order to follow the Playwright API more closely, we are working on migrating most xk6-browser methods to be asynchronous as well.
```